src/favorites.rs

```diff
@@ -42,34 +48,48 @@ pub(crate) fn list_favorites(app_config: &AppConfig, args: &Args) -> Result<()>
     Ok(())
 }
 
-pub(crate) fn resolve_favorite_args(app_config: &AppConfig, args: &mut Args) -> Result<()> {
+pub(crate) fn resolve_favorite_args(
+    app_config: &AppConfig,
+    args: &mut Args,
+) -> Result<Option<HashMap<String, toml::Value>>> {
     if args.git.is_some() {
         args.subfolder = args.favorite.take();
-        return Ok(());
+        return Ok(app_config.values.clone());
     }
 
     let favorite_name = args
         .favorite
         .as_ref()
         .ok_or_else(|| anyhow!("Please specify either --git option, or a predefined favorite"))?;
 
-    let (git, branch, subfolder) = app_config
+    let (values, git, branch, subfolder) = app_config
         .favorites
-        .get(favorite_name.as_str())
+        .as_ref()
+        .and_then(|f| f.get(favorite_name.as_str()))
         .map_or_else(
             || {
-                info!(
+                warn!(
```

Comment:
question: I wonder why that is now a warning? I think a favourite config is still optional, hence I would stick with an info message.

Reply:
hmm, I just renamed the `info!` macro, as it really was outputting a warning, so I didn't change the meaning/output here.

I could however change it, so we just write it as info ??

Reply:
I see, have not recognized it.